### DISCUSSION TITLE
모든 페이지 무한로딩, 클릭 안되는 현상

### DISCUSSION THREAD

**Discussion by signature-hyeon** [pronist/hELLO#167] [Q&A] [answered]
이전에 스킨을 적용하고 잘 작동하는 것도 확인했었는데, 다시 들어가 보니 사진과 같이
**무한 로딩 현상 & 클릭안됨 & 사이드바 사라짐 현상**이 일어납니다.

다른 이슈들 모두 확인해보고 제 환경 및 설정과 비교해보았는데 확인해 본 바로는 모두 동일해 보입니다.
최신 버전 3.6.5 사용 중입니다.
해결하고 싶은데 혼자 힘으로는 무리네요 하하🤠 한번 확인해주시면 감사하겠습니다!

**확인해 본 이슈에 관한 정보**
- adblock 적용되어있지 않습니다.
- 사진과 같이 스킨 설정 시 파일도 잘 들어가 있습니다.
- 삭제했다가 다시 만들어주기도 해보았습니다.

다른 컴퓨터로 접속해보니 해당 이슈들이 없네요 ㅎㅎ **제 로컬 환경 관련 문제 같습니다.**
Chrome이나 Edge에서도 모두 동일한 현상이 일어나는데 무엇이 문제인지 혹시 감이 잡히실까요?
이전에 잘 동작했는데 무엇이 문제인지.. 하핫 궁금합니다.

![image](https://user-images.githubusercontent.com/62706048/211443284-c87ba76c-30f9-45e1-bdd6-17996cae680f.png)

![image](https://user-images.githubusercontent.com/62706048/211443101-3653bc68-b34c-4144-9377-e84b499b0f68.png)

**Comment by pronist** [comment, marked as answer]
@sa46lll 안녕하세요. 일단은 제가 접속해도 정상적으로 접속이 되는 것으로 보입니다.

저 현상이 발생하는 것은 어떤 요소(플러그인과 같은)가 스크립트 로드를 막아버렸기 때문입니다.
크롬에서 사용하시는 플러그인 중, 마크업을 직접 변경할 수 있는 플러그인을 OFF 하시고 접속해보시기 바랍니다.
애드블록도 그 예 중 하나입니다.

**Comment by signature-hyeon** [comment]
모든 크롬 계정의 플러그인을 off 처리하고 캐시도 지우고 온갖 방법을 다 써봐도 안 되더라구요.
알고 보니 AdGuard 프로그램이 실행 중이었습니다 🤣

Edge에서도 안되어 해결책이 너무 궁금했는데, 답변 주신 덕분에 확신을 갖고 해결책을 찾은 것 같습니다.
빠른 답변 감사드리며, 새해 복 많이 받으세요! 스킨 잘 쓰겠습니다!!
